mpack-instance-manager/src/main/python/instance_manager/mpack-instance-manager.py

```diff
@@ -119,7 +121,8 @@ def init_get_parser_options(parser):
   parser.add_option('--components-map', default=None,
                     help="map of 'component type' (eg: hive_server, metastore etc) as key and List of component instance name(s) to be given (eg: HS-1, finance_metastore) as value OR Empty map for all component instances present",
                     dest="components_map")
-
+  parser.add_option('-f', '--format', action="store_true",
```

Comment:
The desired usage is --format=json or --format=simple/plain/readable (by default the command line should output in simple/plain/readable format). Ambari will directly use the mpack-instance-manager python library and not use the command line and will always read in structured json format.